### PR TITLE
Modified how we call mxnet-model-server in container

### PR DESCRIPTION
**Pull request by vdantu**
*Issue #, if available:*

*Description of changes:*
I have modified the way we call mxnet-model-server inside container runs. I have overloaded the existing mxnet-model-server (standalone way of calling). So, when we call "mxnet-model-server" in docker, it calls the containerized (nginx, gunicorn, flask) version of mxnet-model-server scripts. Also added a timeit util to time the methods and added a default config in the container.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by vdantu** [comment]
@aaronmarkham @lupesko @jesterhazy

File: docker/README.md [hunk 1]
```diff
@@ -34,26 +34,28 @@ Docker images are currently available on Docker Hub [CPU](https://hub.docker.com
    ``` 
    Verify the models and configuration file is in this newly created directory. Also verify that the `mms_app_[cpu|gpu].conf` is updated.
 
-3. Run the downloaded Docker image
+3. Run the downloaded CPU container image
    ```bash
    docker run --name mms -p 80:8080 -itd -v /home/users/models/:/models awsdeeplearningteam/mms_cpu
    0c1862a2c30edd0f33391f7277b2c8bcba4a5f4cf22669f3b308b4078a5d3ce8
    ```
+   Note, to run GPU image refer [this section](#running-the-mms-gpu-docker)
```
Comment (lupesko) [inline docker/README.md]: Suggested rephrase: "...refer to [this section]..."

File: docker/README.md [hunk 2]
```diff
@@ -34,26 +34,28 @@ Docker images are currently available on Docker Hub [CPU](https://hub.docker.com
...
    
    Verify the image is running
    ```bash
    docker ps -a
    CONTAINER ID        IMAGE               COMMAND             CREATED                  STATUS              PORTS                NAMES
    b4bab087f2a8        mms_cpu             "/bin/bash"         Less than a second ago   Up 2 seconds        0.0.0.0:80->80/tcp   mms
    ```
+   Refer [Docker CLI](https://docs.docker.com/engine/reference/commandline/run/) to understand each parameter.
```
Comment (lupesko) [inline docker/README.md]: Refer to

File: docker/README.md [hunk 3]
```diff
@@ -85,7 +87,7 @@ If you haven't already, clone the MMS repo and go into the `docker` folder.
 git clone https://github.com/awslabs/mxnet-model-server.git && cd mxnet-model-server/docker
 ```
 
-### Building the Docker Image
+### Building the Container Image
```
Comment (lupesko) [inline docker/README.md]: I don't think we need capitalization here, so: "container image"

File: docker/README.md [hunk 4]
```diff
@@ -357,31 +350,31 @@ curl -X GET http://your_public_host_name/ping
...
 Usage:
 
-/mxnet_model_server/mxnet-model-server.sh [start | stop | restart | help] [--mms-config <MMS config file>]
+mxnet-model-server [start | stop | restart | help] [--mms-config <MMS config file>]
 
 start        : Start a new instance of MXNet model server
 stop         : Stop the current running instance of MXNet model server
 restart      : Restarts all the MXNet Model Server worker instances
-help         : Usage help for /mxnet_model_server/mxnet-model-server.sh
+help         : Usage help for /mxnet_model_server/mms
```
Comment (lupesko) [inline docker/README.md]: Isn't this supposed to be /mxnet-model-server and not /mms?

File: examples/lstm_ptb/README.md [hunk 5]
```diff
@@ -89,21 +89,21 @@ curl -X POST http://127.0.0.1:8080/lstm_ptb/predict -F "data=[{'input_sentence':
 
 Prediction result will be:
 
-```
+```bash
```
Comment (lupesko) [inline examples/lstm_ptb/README.md]: shouldn't this be ```json and not ```bash?

File: examples/lstm_ptb/README.md [hunk 6]
```diff
@@ -89,21 +89,21 @@ curl -X POST http://127.0.0.1:8080/lstm_ptb/predict -F "data=[{'input_sentence':
...
 curl -X POST http://127.0.0.1:8080/lstm_ptb/predict -F "data=[{'input_sentence': 'while friday \'s debacle involved mainly professional traders rather than investors it left the market vulnerable to continued selling this morning traders said '}]"
 ```
 
 Prediction result will be:
 
-```
+```bash
```
Comment (lupesko) [inline examples/lstm_ptb/README.md]: json and not bash?

File: mms/model_service/mxnet_model_service.py [hunk 7]
```diff
@@ -17,7 +17,7 @@
 import json
 import shutil
 import os
-
+import time
```
Comment (lupesko) [inline mms/model_service/mxnet_model_service.py]: Is time needed in this file?

File: docker/README.md (hunk 1 above)
Comment (vdantu) [reply docker/README.md]: Done.

File: docker/README.md (hunk 2 above)
Comment (vdantu) [reply docker/README.md]: done

File: docker/README.md (hunk 3 above)
Comment (vdantu) [reply docker/README.md]: Done

File: docker/README.md (hunk 4 above)
Comment (vdantu) [reply docker/README.md]: True. I missed this. Redid this.

File: examples/lstm_ptb/README.md (hunk 5 above)
Comment (vdantu) [reply examples/lstm_ptb/README.md]: done

File: examples/lstm_ptb/README.md (hunk 6 above)
Comment (vdantu) [reply examples/lstm_ptb/README.md]: done

File: mms/model_service/mxnet_model_service.py (hunk 7 above)
Comment (vdantu) [reply mms/model_service/mxnet_model_service.py]: We don't need this. I removed it.

**Pull request merged**